zeth_contracts/contracts/bls12_377_groth16.sol

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+
+library bls12_377_groth16
```

Comment:
naming style inconsistent, CamelCase is used for other contracts and library code

Reply:
renamed to `Groth16BLS12_377`